python/qpid_dispatch/management/qdrouter.json

```diff
@@ -633,7 +633,13 @@
                     "default": "both",
                     "description": "['in', 'out', 'both', 'no'] in: Strip the dispatch router specific annotations only on ingress; out: Strip the dispatch router specific annotations only on egress; both: Strip the dispatch router specific annotations on both ingress and egress; no - do not strip dispatch router specific annotations",
                     "create": true
-                }
+                },
+                "protocolFamily": {
```

Comment:
Rather than duplicate this entry for both listener and connector, you could put this in the "addrPort" annotation.  Like addr and port, protocolFamily is common to both entity types.

You have a default of IPv4 in the connector.  That is inconsistent with the document string.